src/scope.js

```diff
@@ -21,14 +21,15 @@
   (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
   THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 */
+"use strict";
 
-import { Syntax } from 'estraverse';
-import Map from 'es6-map';
+const Syntax = require('estraverse').Syntax;
+const Map = require('es6-map');
```

Comment:
I think this import is unnecessary now since `Map` is included in Node 4. (That can be changed in a separate PR if necessary though.)